src/avram/box.cr

```diff
@@ -36,8 +36,36 @@ abstract class Avram::Box
     operation.save!
   end
 
+  # Returns an array with 2 instances of the model from the Box.
+  #
+  # Usage:
+  #
+  # ```crystal
+  # tags = TagBox.create_pair
+  # typeof(tags) # => Array(Tag)
+  # tags.size    # => 2
+  # ```
+  def self.create_pair
+    [1, 2].map do |n|
+      new.create
+    end
```

Comment:
I think this should call `create_pair { }` so that the implementation is DRYed up a bit. Thoughts?

Reply:
I originally did that and ran in to a weird issue passing the block around. I'd rather go that route though, so I'll give it another shot and post what issue I run in to.